docs/pages/versions/unversioned/sdk/tracking-transparency.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Tracking Transparency
+sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-tracking-transparency'
+---
+
+import APISection from '~/components/plugins/APISection';
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+import SnackInline from '~/components/plugins/SnackInline';
+
+A library for requesting permission to track the user or their device. Examples of data used for tracking include email address, device ID, advertising ID, etc. This permission is only necessary on iOS 14.5 and higher.
+
+For more information on Apple's new App Tracking Transparency framework, please refer to their [documentation](https://developer.apple.com/app-store/user-privacy-and-data-use/).
+
+<PlatformsSection android emulator ios simulator web />
```

Comment:
probably just ios / simulator here right?

Reply:
Yeah good question- I wasn't sure what our stance is here. The module is supported on android and web (you won't get an UnavailabilityError), but it's not doing anything

Reply:
hm yeah i think we don't have explicit expected behavior here.

if we look at [IntentLauncher](https://docs.expo.io/versions/v41.0.0/sdk/intent-launcher/) (an android only module) we can see that it's listed as supporting ios - i thought this could because it has no side effects and executes w/o complaint like here, but it turns out that [it actually throws an error on ios](https://github.com/expo/expo/blob/304270d4c86a95d3475aa064b8ea69a2d1ee7e87/packages/expo-intent-launcher/src/IntentLauncher.ts#L131-L133). the only other comparable module (one that is purpose built for a single platform) is [AppleAuthentication](https://docs.expo.io/versions/v41.0.0/sdk/apple-authentication/) which does not support android in the compat table and also errors on usage.

a pattern that we use in the AppleAuthentication case is to provide an `isAvailableAsync()` function that developers can use to gate their calls. perhaps what we can do here is add that same function to this module, mark the other platforms as unsupported, and throw an exception when attempting to get permissions on unsupported platforms. curious what others think so i will share this in #api too